### ISSUE TITLE
[FR] Re-add default of "extract all" for items put into dank nulls (or add the option as a config setting)

### ISSUE THREAD

**Issue by Phys09** [p455w0rd/DankNull#339]
Allows player to more easily use the dank null as a portable "backpack."

Is a potential pseudo workaround for #327

apparently Version 1.7.97 had this as default

"""
** Version 1.7.97 (Beta) **

Change default extraction mode to keep 1 instead of keep all
"""

**Comment by Zafaranec** [comment]
Yes please, Dank/Null can be best backpack mod but with default on "keep 1" is really painful to use it like that. I know, you change it with updates maybe three times, can you make it configurable through config please...

**Comment by bschultz0** [comment]
Was just searching for a way to change the 'default' mode to extract all.  Apparently there isn't one.  It is very annoying to use this in your storage system and have items disappear from your storage network because they aren't visible due to dank holding one that is not able to extract by default.  Going into each one on each item to change it to allow it to extract all isn't a real workaround as the second you extract it and then put the same type of item back, it defaults to 'all but 1' again.  Very annoying.  The default extraction mode should be configurable.  If not in game, then in the config.
